ffi/src/rdcleanpath.rs

```diff
@@ -0,0 +1,216 @@
+#[diplomat::bridge]
+pub mod ffi {
+    use core::fmt::Write as _;
+    use diplomat_runtime::DiplomatWriteable;
+
+    use crate::error::ffi::IronRdpError;
+    use crate::error::GenericError;
+    use crate::utils::ffi::VecU8;
+
+    #[diplomat::opaque]
+    pub struct RDCleanPathPdu(pub ironrdp_rdcleanpath::RDCleanPathPdu);
+
+    impl RDCleanPathPdu {
+        /// Creates a new RDCleanPath request PDU
+        ///
+        /// # Arguments
+        /// * `x224_pdu` - The X.224 Connection Request PDU bytes
+        /// * `destination` - The destination RDP server address (e.g., "10.10.0.3:3389")
+        /// * `proxy_auth` - The JWT authentication token
+        /// * `pcb` - Optional preconnection blob (for Hyper-V VM connections, empty string if not needed)
+        pub fn new_request(
+            x224_pdu: &[u8],
+            destination: &str,
+            proxy_auth: &str,
+            pcb: &str,
+        ) -> Result<Box<RDCleanPathPdu>, Box<IronRdpError>> {
+            let pcb_opt = if pcb.is_empty() { None } else { Some(pcb.to_owned()) };
+
+            let pdu = ironrdp_rdcleanpath::RDCleanPathPdu::new_request(
+                x224_pdu.to_vec(),
+                destination.to_owned(),
+                proxy_auth.to_owned(),
+                pcb_opt,
+            )
+            .map_err(|e| GenericError(format!("Failed to create RDCleanPath request: {e}")))?;
```

Comment:
Please, use the standard style guidelines for message formatting: https://github.com/Devolutions/IronRDP/blob/master/STYLE.md#formatting-of-error-messages